specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```diff
@@ -0,0 +1,5247 @@
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "EncryptionStatus",
+            "modelAsString": true
+          }
+        },
+        "keyVaultProperties": {
+          "$ref": "#/definitions/KeyVaultProperties",
+          "description": "Customer Key vault properties."
+        }
+      },
+      "required": [
+        "status",
+        "keyVaultProperties"
+      ],
+      "type": "object"
+    },
+    "KeyVaultProperties": {
+      "properties": {
+        "keyVaultArmId": {
+          "description": "The ArmId of the keyVault where the customer owned encryption key is present.",
+          "type": "string"
+        },
+        "keyIdentifier": {
+          "description": "Key vault uri to access the encryption key.",
+          "type": "string"
+        },
+        "identityClientId": {
+          "description": "For future use - The client id of the identity which will be used to access key vault.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "keyIdentifier",
+        "keyVaultArmId"
+      ],
+      "type": "object"
+    },
+    "LinkedWorkspace": {
+      "description": "Linked workspace.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "ResourceId of the link of the linked workspace."
+        },
+        "name": {
+          "type": "string",
+          "description": "Friendly name of the linked workspace."
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type of linked workspace."
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedWorkspaceProps"
+        }
+      }
+    },
+    "LinkedWorkspaceDto": {
+      "type": "object",
+      "description": "object used for creating linked workspace.",
+      "properties": {
+        "name": {
+          "description": "Friendly name of the linked workspace",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/LinkedWorkspaceProps"
+        }
+      }
+    },
+    "LinkedWorkspaceProps": {
+      "type": "object",
+      "description": "LinkedWorkspace specific properties.",
+      "properties": {
+        "linkedWorkspaceResourceId": {
+          "description": "ResourceId of the link target of the linked workspace.",
+          "type": "string"
+        },
+        "userAssignedIdentityResourceId": {
+          "description": "ResourceId of the user assigned identity for the linked workspace.",
+          "type": "string"
+        }
+      }
+    },
+    "ServiceResponseBase": {
+      "description": "The base service response. The correct inherited response based on computeType will be returned (ex. ACIServiceResponse)",
+      "required": [
+        "computeType"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
```

Comment:
This doesn't look like an Azure resource.  Please conform to the basic resource structure defined in https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#put-resource

Reply:
made changes.